tests/e2e_operations.py

```diff
@@ -87,31 +108,65 @@ def test_forced_ledger_chunk(network, args):
     # Check that there is indeed a ledger chunk that ends at the
     # first signature after proposal.completed_seqno
     ledger = ccf.ledger.Ledger(ledger_dirs)
-    for chunk in ledger:
-        first = last = next_signature = None
-        for tx in chunk:
-            pd = tx.get_public_domain()
-            if first is None:
-                first = pd.get_seqno()
-            else:
-                last = pd.get_seqno()
-            tables = pd.get_tables()
-            if (
-                pd.get_seqno() >= proposal.completed_seqno
-                and next_signature is None
-                and ccf.ledger.SIGNATURE_TX_TABLE_NAME in tables
-            ):
-                next_signature = pd.get_seqno()
-        if first <= proposal.completed_seqno and proposal.completed_seqno <= last:
-            LOG.info(
-                f"Found ledger chunk {chunk.filename()} with chunking proposal @{proposal.completed_seqno} and signature @{next_signature}"
-            )
-            assert last == next_signature
-            assert next_signature - proposal.completed_seqno < args.sig_tx_interval
-
+    chunk, _, last, next_signature = find_ledger_chunk(ledger, proposal.completed_seqno)
+    LOG.info(
+        f"Found ledger chunk {chunk.filename()} with chunking proposal @{proposal.completed_seqno} and signature @{next_signature}"
+    )
+    assert chunk.is_complete and chunk.is_committed()
+    assert last == next_signature
+    assert next_signature - proposal.completed_seqno < args.sig_tx_interval
     return network
 
 
+@reqs.description("Forced snapshot")
+def test_forced_snapshot(network, args):
+    primary, _ = network.find_primary()
+
+    # Submit some dummy transactions
+    network.txs.issue(network, number_txs=7)
+
+    # Submit a proposal to force a snapshot at the following signature
+    proposal_body, careful_vote = network.consortium.make_proposal(
+        "request_snapshot", node_id=primary.node_id
+    )
+    proposal = network.consortium.get_any_active_member().propose(
+        primary, proposal_body
+    )
+
+    proposal = network.consortium.vote_using_majority(
+        primary,
+        proposal,
+        careful_vote,
+    )
+
+    # Issue some more transactions
+    network.txs.issue(network, number_txs=13)
+
+    # Wait for the signature interval to ensure we see at least one signature
+    time.sleep(args.sig_ms_interval / 1000)
```

Comment:
`txs.issue()` already waits for a signature to be emitted (since it waits for commit of the transactions) so this shouldn't be necessary here?